com.unity.render-pipelines.universal/Runtime/Passes/AdditionalLightsShadowCasterPass.cs

```diff
@@ -786,10 +788,19 @@ public bool Setup(ref RenderingData renderingData)
             m_AdditionalLightsShadowmapTexture = ShadowUtils.GetTemporaryShadowTexture(renderTargetWidth, renderTargetHeight, k_ShadowmapBufferBits);
             m_MaxShadowDistanceSq = renderingData.cameraData.maxShadowDistance * renderingData.cameraData.maxShadowDistance;
             m_CascadeBorder = renderingData.shadowData.mainLightShadowCascadeBorder;
+            m_CreateEmptyShadowmap = false;
+            useNativeRenderPass = true;
 
             return true;
         }
 
+        internal void SetupEmpty()
```

Comment:
Can we improve the naming of this function to be a bit more descriptive?

Reply:
SetupForEmptyRendering? Quite long :D